### PR TITLE
Added rubygem-newt

### PR DESCRIPTION
**Pull request by lzap**
Replaces https://github.com/theforeman/foreman-packaging/pull/687

### CODE REVIEW

File: rel-eng/tito.props [hunk 1]
```diff
@@ -23,6 +23,7 @@ blacklist = foreman-installer
   rubygem-clamp
   rubygem-hammer_cli
   rubygem-hammer_cli_foreman
+  rubygem-newt
```
Comment (domcleal) [inline rel-eng/tito.props]: No need to add it to non-plugin releasers?

File: comps/comps-foreman-fedora19.xml [hunk 2]
```diff
@@ -76,6 +76,7 @@
       <packagereq type="default">rubygem-mysql2</packagereq>
       <packagereq type="default">rubygem-net-ldap</packagereq>
       <packagereq type="default">rubygem-net-scp</packagereq>
+      <packagereq type="default">rubygem-newt</packagereq>
```
Comment (domcleal) [inline comps/comps-foreman-fedora19.xml]: Wrong file, plugins have their own comps.

**Comment by lzap** [comment]
Pushed, I am unable to submit scratch builds tho:

```
tito build --test koji-foreman-plugins-nightly
WARNING: unable to lookup latest package tag, building untagged test project
WARNING: rel-eng/packages/rubygem-newt doesn't exist in git, using current directory
Building package [rubygem-newt-0.9.4-1]
```

**Comment by lzap** [comment]
http://koji.katello.org/koji/taskinfo?taskID=291466
http://koji.katello.org/koji/taskinfo?taskID=291465
http://koji.katello.org/koji/taskinfo?taskID=291469

File: README.md [hunk 3]
```diff
@@ -51,11 +51,11 @@ than mock (above).
 
 Non-core packages in the main Foreman repo, with sources in git(-annex):
 
-    tito release koji-foreman --test --scratch
+    tito release --test --scratch koji-foreman
 
 Packages in the plugins repo:
 
-    tito release koji-foreman-plugins --test --scratch
+    tito release --test --scratch koji-foreman-plugins
```
Comment (domcleal) [inline README.md]: Revert these.. they're the same.

File: comps/comps-foreman-plugins-rhel6.xml [hunk 4]
```diff
@@ -49,6 +49,7 @@
       <packagereq type="default">rubygem-smart_proxy_openscap</packagereq>
       <packagereq type="default">rubygem-smart_proxy_pulp</packagereq>
       <packagereq type="default">rubygem-smart_proxy_salt</packagereq>
+      <packagereq type="default">rubygem-newt</packagereq>
```
Comment (domcleal) [inline comps/comps-foreman-plugins-rhel6.xml]: Move this up a bit, it's alphabetical

File: comps/comps-foreman-plugins-rhel7.xml [hunk 5]
```diff
@@ -50,6 +50,7 @@
       <packagereq type="default">rubygem-smart_proxy_chef</packagereq>
       <packagereq type="default">rubygem-smart_proxy_pulp</packagereq>
       <packagereq type="default">rubygem-smart_proxy_salt</packagereq>
+      <packagereq type="default">rubygem-newt</packagereq>
```
Comment (domcleal) [inline comps/comps-foreman-plugins-rhel7.xml]: ditto

File: rubygem-newt/rubygem-newt.spec [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+Version: 0.9.4
+Release: 1%{?dist}
+Group: Development/Languages
+License: MIT
+URL: https://github.com/theforeman/ruby-newt
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+Requires: ruby
+Requires: rubygems
+
```
Comment (domcleal) [inline rubygem-newt/rubygem-newt.spec]: remove blank

File: rubygem-newt/rubygem-newt.spec [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+Requires: newt
+
+BuildRequires: ruby
+BuildRequires: rubygems
+BuildRequires: rubygems-devel
+BuildRequires: ruby-devel
+
```
Comment (domcleal) [inline rubygem-newt/rubygem-newt.spec]: ditto

File: rubygem-newt/rubygem-newt.spec [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+%build
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+mkdir -p %{buildroot}%{gem_extdir_mri_lib}/ruby_newt
+mv %{buildroot}%{gem_instdir}/lib/ruby_newt/ruby_newt.so %{buildroot}%{gem_extdir_mri_lib}/ruby_newt/ruby_newt.so
+rm -rf %{buildroot}%{gem_dir}/gems/%{gem_name}-%{version}/{ext,tmp,.require_paths}
```
Comment (domcleal) [inline rubygem-newt/rubygem-newt.spec]: `%{gem_dir}/gems/%{gem_name}-%{version}` == `%{gem_instdir}` or just use %exclude in the files list

**Comment by lzap** [comment]
Done, let me test them again on EL systems.

**Comment by lzap** [comment]
Made a typo, once again.

**Comment by lzap** [comment]
Works on both RHELs.

**Comment by domcleal** [comment]
Thanks @lzap, will build to nightly plugins.

**Comment by lzap** [comment]
Thanks for patience with me :-)

**Comment by lzap** [comment]
Will that hit 1.9? Would like to introduce the TUI for 1.9 Discovery...

**Comment by domcleal** [comment]
Sure, we can retag plugin packages into whatever release repo necessary.  I'll retag into 1.9.

**Pull request merged**